scio-test/parquet/src/main/scala/com/spotify/scio/testing/parquet/avro/package.scala

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2024 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.testing.parquet
+
+import com.spotify.scio.testing.parquet.ParquetTestUtils._
+import magnolify.parquet.ParquetType
+import org.apache.avro.Schema
+import org.apache.avro.generic.GenericRecord
+import org.apache.hadoop.conf.Configuration
+import org.apache.parquet.avro.{AvroParquetReader, AvroParquetWriter, AvroReadSupport}
+import org.apache.parquet.filter2.predicate.FilterPredicate
+import org.apache.parquet.hadoop.{ParquetInputFormat, ParquetReader}
+import org.apache.parquet.io.InputFile
+
+package object avro {
+  implicit def toParquetAvroHelpers[U <: GenericRecord](
+    records: Iterable[U]
+  ): ParquetAvroHelpers[U] = new ParquetAvroHelpers(records)
+
+  class ParquetAvroHelpers[U <: GenericRecord] private[testing] (
+    records: Iterable[U]
+  ) {
```

Comment:
Nit: we should avoid using package object for definitions as it is deprecated in scala 3 ([ref](https://docs.scala-lang.org/scala3/reference/dropped-features/package-objects.html)),